### ISSUE TITLE
svg enhancement

### ISSUE THREAD

**Issue by Neon22** [pyscript/ltk#5]
I was trying to change the width of an svg rect I'd made.:
Made it like:
```
SVG.rect(x=self.draft_handle.parent.start[0], y=self.draft_handle.parent.start[1],
      width=width, height=height,
      style=style, id="baserect")
```
Its all good and onscreen. Then I attempt to resize it using:
`ltk.find("#baserect").attr("width", width)`

But no go for reasons below.

However it can be done by using the regular dom:
`ltk.window.document.querySelector("#baserect").setAttribute('width', width)`

Jquery has uneven support for SVG. (There is an svg extension for jquery though which make some overrides. Not sure if its in Jquery 3 by default).
I note that when attempting to modify (say) the width of an svg rectangle (using its id), you get:
an object of type = `SVGAnimatedLength` with roughly this structure:
```
baseVal: {
    unitType: number,
    value: number,
    valueInSpecifiedUnits: number,
    valueAsString: number
},
animVal: { /* etc */ }
```

Clearly using this could be good because we could animate with it.
I'm not sure if ALL browsers return the same thing but can the ltk take advantage if it does ??

Halfway down this page is some discussion on this class and Jquery3
- https://stackoverflow.com/questions/52958178/why-jquery-cannot-modify-svg-element-width-height-tag-attributes-but-works-fine

**Comment by laffra** [comment]
SVG objects are read-only, so you cannot change the width/height after construction.

**Comment by Neon22** [comment]
Yes - at the moment I tend to completely replace the svg if I want to change some aspect of it.
Probably this should be closed. I was hoping the jQuery extension would allow me to avoid using my special `svg.py` add in but this is probably a mistake.
Animation is a nice to have but not essential

**Comment by laffra** [comment]
You can probably get size animation by setting the CSS "scale"?
